app/src/main/java/org/fossasia/susi/ai/rest/clients/DeviceClient.java

```diff
@@ -4,7 +4,7 @@
 import retrofit2.converter.gson.GsonConverterFactory;
 
 public class DeviceClient {
-    public static final String SUSI_SPEAKER_BASE_URL = "http://192.168.0.5:5000";
+    public static final String SUSI_SPEAKER_BASE_URL = "http://10.0.0.1:5000";
```

Comment:
This cannot be a constant

Reply:
@iamareebjamal okay,
also please give a review of this PR